src/dos/cdrom_image.cpp

```diff
@@ -313,58 +549,140 @@ bool CDROM_Interface_Image::ReadSector(Bit8u *buffer, bool raw, unsigned long se
 	if (tracks[track].sectorSize == RAW_SECTOR_SIZE && !tracks[track].mode2 && !raw) seek += 16;
 	if (tracks[track].mode2 && !raw) seek += 24;
 
+	// LOG_MSG("CDROM: ReadSector track=%d, desired raw=%s, sector=%ld, length=%d", track, raw ? "true":"false", sector, length);
 	return tracks[track].file->read(buffer, seek, length);
 }
 
+void printProgress(double percentage, const char* msg)
+{
+	// 60 is the number of characters in the full progress bar
+	int val  = (int)(percentage * 100);
+	int lpad = (int)(percentage * 60);
+	int rpad = 60 - lpad;
+	LOG_MSG("\r%3d%% [%.*s%*s] - %s", val, lpad,
+	   "||||||||||||||||||||||||||||||||||||||||||||||||||||||||||||", rpad, "", msg);
+	fflush(stdout);
+}
```

Comment:
Is this function still useful?

Reply:
Knocked it out along with the commented-out calls to it; it's past its usefulness.